tasks/windows.ps1

```diff
@@ -188,6 +188,9 @@ function Invoke-SimplifiedInstaller
 
 try
 {
+  # Enable TLS 1.2 to allow connection to the master
+  [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Is this way of setting the SecurityProtocol preferred over the way it is done above https://github.com/puppetlabs/puppetlabs-bootstrap/blob/a7c42f4d2403713973ad0edef250d87a0f32af98/tasks/windows.ps1#L62

If so, can we change it there as well for consistency's sake?